Formula/haproxy.rb

```diff
@@ -21,16 +21,23 @@ class Haproxy < Formula
   depends_on "pcre"
 
   def install
+    target = "generic"
+    on_linux do
+      target = "linux-glibc"
+    end
     args = %w[
-      TARGET=generic
-      USE_KQUEUE=1
+      TARGET=#{target}
```

Comment:
Does this work on Linux? Because you're using `%w`, which does not allow for interpolation, so actually `TARGET` is being set to garbage at the moment.

Reply:
Woups sorry. I changed the code. Nice catch.